packages/fx-core/src/core/middleware/projectUpgrader.ts

```diff
@@ -115,12 +115,18 @@ export async function upgradeContext(ctx: CoreHookContext): Promise<void> {
     });
   } catch (error) {
     const errorObject = ContextUpgradeError(error);
+    core.tools.logProvider.info(
```

Comment:
![image](https://user-images.githubusercontent.com/16380704/127586510-9295102f-7dc3-4829-9478-af77f8a7b3ba.png)
need to add ? to check undefine case.

Reply:
Fixed. Thanks!